### ISSUE TITLE
data url

### ISSUE THREAD

**Issue by basz** [rtablada/ember-css-properties#4]
This line https://github.com/rtablada/ember-css-properties/blob/master/addon/helpers/css-properties.js#L15 makes it impossible to add a value such as;

`url('data:image/png;base64,iVBORw0KGgoAAAANSUhEU...')`

Am I missing something? What's the reason for that `;` split anyway?

a `;` is valid in data urls' https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs

**Comment by richard-viney** [comment]
Ran into this yesterday, definitely unexpected not to be able to use with a data URL.

**Comment by richard-viney** [comment]
Looks like @basz has already attended to it on this fork:  https://github.com/basz/ember-css-properties/commits/remove-split - any chance of merging this in?

**Comment by richard-viney** [comment]
Bump - do we just need this to be PR'd.. ?

**Comment by richard-viney** [comment]
After some playing around with this I ended up creating an `inline-styles` modifier rather than using the `css-properties` helper in this addon.

This approach means that the code assigns directly to individual values on the `DOMElement.style` object, which appears to avoid the troubles with needing to escape the provided values to protect against CSS injections as discussed in #5, which currently means data URLs can't be used with this addon.

An experimental addon implementing the `inline-styles` modifier can be found at https://github.com/richard-viney/ember-inline-styles-modifier

Any thoughts on this alternative approach or other drawbacks to it I'm not aware of?

Thanks.

**Comment by buschtoens** [comment]
Were you aware of https://github.com/jelhan/ember-style-modifier ?

**Comment by richard-viney** [comment]
Yes I saw that after writing my little modifier :-p. I had been meaning to learn how to publish an Ember addon, so it was more an excuse to finally do that.

I'm still curious as to whether the modifier approach is now generally seen as the way to go rather than using the string building/concat approach of this addon.
